my-app/src/components/login/LogInComponent.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import authService from "../../services/auth.service";
+import { Redirect } from "react-router-dom";
+
+class LogIn extends Component {
+  state = {
+    isAuthorized: authService.isAuthorized(), // on init
+    error: null,
+    isLoaded: false,
+    username: "",
+    password: "",
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+
+    authService
+      .login({
+        username: this.state.username,
+        password: this.state.password,
+      })
+      .then((res) => {
+        this.setState({ isAuthorized: authService.isAuthorized() });
+      })
+      .catch((error) => {
+        alert(error.message);
+      });
+  };
+
+  onChangeInput = (e) => {
+    const name = e.target.name;
+    this.setState({ [name]: e.target.value });
```

Comment:
If you put name as variable, probably need also put e.target.value to variable to have the same code style